python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -970,7 +969,9 @@ def setDataUnitRegion(self, dataUnitNames, value, region, update=True):
         ----------
         dataUnitNames : sequence
             A sequence of DataUnit names whose instances are jointly associated
-            with a region on the sky.
+            with a region on the sky. This should not include dependencies that
+            are implied, e.g. "Patch" should not include "Tract", but "Sensor"
+            should add "Visit".
```

Comment:
Is this "should not" actually a "need not" or a "must not"?  I'm a bit worried that in some contexts it may be difficult to *remove* dependendencies if they are already present.

Reply:
For now, it's probably best to just change the comment to "must not" and get the bugfix merged.  This is probably best addressed on or after DM-15034.